deployability/README.MD

```diff
@@ -299,18 +299,18 @@ Deployability contains the following directories:
 
 #### Overview
 
-![image](https://github.com/wazuh/wazuh-qa/assets/125690423/d09dfa61-67cb-410d-8677-bd904d5c4d51)
+![image](https://github.com/wazuh/wazuh-qa/assets/2949519/f1a11353-d61f-4afe-9290-b3c33f619f60)
 
 #### Allocator, provision & test process
 
-![image](https://github.com/wazuh/wazuh-qa/assets/125690423/b1e24a43-13d7-4035-a3e5-5ace4667ec10)
+![image](https://github.com/wazuh/wazuh-qa/assets/2949519/ca927c80-dcde-4b6d-a72e-9c5b6ed601df)
 
 
 
 ----
 
 
-[draw-plot.zip](https://github.com/wazuh/wazuh-qa/files/14330528/plot.zip)
+[draw-plot.zip](https://github.com/user-attachments/files/15778407/draw-plot.zip)
```

Comment:
JobFlow

Reply:
fixed in 
[draw-plot.zip](https://github.com/user-attachments/files/15792115/draw-plot.zip)